CHANGELOG.md

```diff
@@ -23,3 +23,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Created base image for sentiment analysis with model [#55](https://github.com/ncsa/standalone-smm-analytics/issues/55)
 - Created base image for name entity recognition with model [#56](https://github.com/ncsa/standalone-smm-analytics/issues/56)
 - Docker compose file updated to fix minio default bucket making [#63](https://github.com/ncsa/standalone-smm-analytics/issues/63)
+- Reorganized repository structure [#10](https://github.com/ncsa/standalone-smm-analytics/issues/10)
```

Comment:
Can you start a Unrelease session. Last release is Beta release, i'm thinking of proper tagging it from now on

Reply:
oh, didn't know. sorry, updated